Makefile

```diff
@@ -284,14 +284,16 @@ test-e2e:
 	go test ./test/e2e/generate -v
 	$(eval export E2E="")
 
-test-e2e-local:
+port-forward.PID:
+	kubectl port-forward -n kyverno service/kyverno-svc-metrics 8000:8000 & echo $$! > $@
+
+test-e2e-local: port-forward.PID
```

Comment:
This will cause the port forward to happen before the e2e env var check which means it is possible that the port forward is not cleaned up. We should move the env var check to a guard and set that as a pre requisite to this make target.